zgl.zig

```diff
@@ -1433,3 +1373,7 @@ pub fn checkFramebufferStatus(target: FramebufferTarget) FramebufferStatus {
 pub fn drawBuffers(bufs: []const FramebufferAttachment) void {
     c.glDrawBuffers(cs2gl(bufs.len), @ptrCast([*]const c.GLuint, bufs.ptr));
 }
+
+pub fn hasExtension(extension: [:0]const u8) bool {
```

Comment:
Please rewrite this to not use libepoxy here. I want to remove the requirement for epoxy and make this thing a standalone library for zig one day.

You can query the amount and extensions via the respective calls to `glGet` and `glGetString` (there's an index variant)